app/models/item.rb

```diff
@@ -1,3 +1,9 @@
 class Item < ApplicationRecord
   belongs_to :supermarket
+  has_many :customer_items
+  has_many :customers, through: :customer_items
+
+  def number_sold
+    self.customers.count
+  end
```

Comment:
Nice job creating this method. No need for the self, can just put `customers.count` inside this method.